### PR TITLE
Multiple coupon codes on a Subscription

### LINKED ISSUE #145: Subscriptions only allow you to add one coupon code
There's only one method "setCouponCode" on a subscription. However, Recurly allows you to add 
[multiple coupons for a subscription](https://docs.recurly.com/docs/coupons).

### PR DESCRIPTION
**Pull request by bhelx**
Resolves #145

### CODE REVIEW

File: src/main/java/com/ning/billing/recurly/model/Subscription.java [hunk 1]
```diff
@@ -385,6 +400,8 @@ public String toString() {
         sb.append(", unitAmountInCents=").append(unitAmountInCents);
         sb.append(", currency='").append(currency).append('\'');
         sb.append(", quantity=").append(quantity);
+        sb.append(", couponCode=").append(couponCode);
```
Comment (pierre) [inline src/main/java/com/ning/billing/recurly/model/Subscription.java]: `equals`/`hashCode` might need to be updated too?

File: src/test/java/com/ning/billing/recurly/model/TestSubscription.java [hunk 2]
```diff
@@ -81,6 +85,7 @@ public void testDeserialization() throws Exception {
                                         "</subscription>";
 
         final Subscription subscription = verifySubscription(subscriptionData);
+        System.out.println(subscription.getCouponCodes());
```
Comment (pierre) [inline src/test/java/com/ning/billing/recurly/model/TestSubscription.java]: Extra debugging?

**Comment by bhelx** [comment]
@pierre okay I've updated. Thanks for taking a look!

**Comment by pierre** [comment]
👍

**Pull request merged**